go-controller/pkg/ovn/ovn.go

```diff
@@ -58,6 +58,19 @@ type ACLLoggingLevels struct {
 	Deny  string `json:"deny,omitempty"`
 }
 
+type sharedPortGroupInfo struct {
+	sync.Mutex
+	pgName           string
+	policies         map[string]bool // policies sharing this port group
+	lspIngressRefCnt int
```

Comment:
what is this for? can you add some comments here?

Reply:
sure, will add